lib/approval/service.rb

```diff
@@ -0,0 +1,32 @@
+require 'approval-api-client-ruby'
+
+module Approval
+  class Service
+    def self.approval_api
```

Comment:
This can probably be a private method with `klass` as an argument and then we can just pass that through on line 12 instead of having to do `@klass = klass` on line 10.

This would force all uses of `Approval::Service` to the `.call` method, which is all we're using anyway.

Reply:
Refactored it to look more like `lib/rbac/service.rb` so that we can get around this. Before I had modelled it around `lib/topological_inventory.rb` which isn't really comparable since thats only one API class.